docs/Reply.md

```diff
@@ -5,6 +5,7 @@
   - [Introduction](#introduction)
   - [.code(statusCode)](#codestatuscode)
   - [.statusCode](#statusCode)
+  - [.instance](#instance)
```

Comment:
I think `.instance` is going to be confusing. Is it the instance of the server or the instance of the request/reply object? It's ambiguous. What do we think about `.server` instead?

Reply:
I prefer .server as well